G3D10/G3D-app.lib/source/LightingEnvironment.cpp

```diff
@@ -149,6 +150,19 @@ LightingEnvironment::LightingEnvironment(const Any& any) {
 
     r.getIfPresent("ambientOcclusionSettings", ambientOcclusionSettings);
 
+    ddgiVolumeArray.fastClear();
+    ddgiVolumeSpecificationArray.fastClear();
+    // Irradiance volume
+    if (any.containsKey("ddgiVolumeArray")) {
+        Any ddgiVolumes = any["ddgiVolumeArray"];
+        if (ddgiVolumes.size() > 0) {
+            for (Any::AnyTable::Iterator it = ddgiVolumes.table().begin(); it.isValid(); ++it) {
+                ddgiVolumeSpecificationArray.append(DDGIVolumeSpecification(it->value));
+            }
+        }
+        r.setReadStatus("ddgiVolumeArray", true);
+    }
+
```

Comment:
I notice this same code happens in `DefaultRenderer::createProbeVolumes` which then copies the `DefaultRenderer::m_ddgiVolumeArray` into `LightingEnvironment::ddgiVolumeArray`.

I assume this code path is for loading from Any while the other is for setting up volumes when they may not have been specified by an Any. I assume we want the Any specification to have priority. Is this priority enforced?